hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/TestStateContext.java

```diff
@@ -190,46 +197,6 @@ public void testReportQueueWithAddReports() {
 
     Map<String, Integer> expectedReportCount = new HashMap<>();
 
-    // Add a bunch of ContainerReports
```

Comment:
Is it related to remove these test cases?

Reply:
I deleted these tests because they are already covered in a different method(testGetReports()) in the same file. This is to avoid redundant testing.